This patch complete uses the tool "install" to install
all files and to create the directories. Furthermore it also
adds the posibility to place all the files into a DESTDIR -->
this is usefull for packaging --> as most distributers do

--- nonsequencer-0.1/Makefile.orig	2008-08-12 04:18:43.000000000 +0200
+++ nonsequencer-0.1/Makefile	2008-09-05 12:42:35.000000000 +0200
@@ -108,15 +108,16 @@
 
 install: all
 	@ echo -n "Installing..."
-	@ install non-sequencer $(prefix)/bin
-	@ mkdir -p "$(SYSTEM_PATH)"
-	@ cp -r instruments "$(SYSTEM_PATH)"
-	@ mkdir -p "$(DOCUMENT_PATH)"
-	@ cp doc/*.html doc/*.png "$(DOCUMENT_PATH)"
+	@ install -d $(DESTDIR)$(prefix)/bin
+	@ install -m 755 non-sequencer $(DESTDIR)$(prefix)/bin
+	@ install -d "$(DESTDIR)$(SYSTEM_PATH)/instruments"
+	@ install -m 644 instruments/* "$(DESTDIR)$(SYSTEM_PATH)/instruments"
+	@ install -d "$(DESTDIR)$(DOCUMENT_PATH)"
+	@ install -m 644 doc/*.html doc/*.png "$(DESTDIR)$(DOCUMENT_PATH)"
 	@ echo "$(DONE)"
 ifneq ($(USE_DEBUG),yes)
 	@ echo -n "Stripping..."
-	@ strip $(prefix)/bin/non-sequencer
+	@ strip $(DESTDIR)$(prefix)/bin/non-sequencer
 	@ echo "$(DONE)"
 endif
 
